protokube/pkg/protokube/volume_mounter.go

```diff
@@ -52,6 +52,12 @@ func (k *VolumeMountController) mountMasterVolumes() ([]*Volume, error) {
 	}
 
 	for _, v := range attached {
+		// Do not ever try to mount root devices
+		if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
```

Comment:
Is this the meat of the change?  It's quite surprising to me that these volumes got through the etcd tag filters ...